Curve-finance-suite/src/tests/Curve-Dao-ClaimMany.spec.ts

```diff
@@ -0,0 +1,53 @@
+import {
+  Finding,
+  HandleTransaction,
+  createTransactionEvent,
+} from "forta-agent";
+import providesetRewardsAgent, {
+  web3,
+  setRewards,
+} from "../agents/Curve-Gauge-SetRewards";
+
+const ADDRESS = "0x1111";
+const PAYOUTADDRESS = "0x5C34E725CcA657F02C1D81fb16142F6F0067689b";
+const ALERTID = "NETHFORTA-21-7";
+
+describe("high gas agent", () => {
```

Comment:
Change the `describe` name.

Also add more test, to check other cases returning 0 findings and cases when the function is called in internal txns